rest/api_collections_test.go

```diff
@@ -464,6 +464,28 @@ func TestCollectionsSGIndexQuery(t *testing.T) {
 	_, err := rt.WaitForChanges(1, "/{{.keyspace}}/_changes", username, false)
 	require.NoError(t, err)
 }
+func TestCollectionsPutDBInexistentCollection(t *testing.T) {
+	base.TestRequiresCollections(t)
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll)
+
+	if !base.TestUseXattrs() {
+		t.Skip("Test relies on import - needs xattrs")
+	}
+
```

Comment:
Are these (I noticed the same in other PRs) coming from copy/paste of existing tests or somewhere else? We should try to minimise and simplify tests where possible.

This test doesn't write to the bucket/use import. The vast majority of tests won't use import as they do their document writes through Sync Gateway, and not Couchbase Server's bucket.

```suggestion
```

Reply:
https://github.com/couchbase/sync_gateway/pull/6018/commits/19ea492a3a4bafab3e78c85a81fffe8f2e122287